### PR TITLE
Task/14281 Adapt thank you screen

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Adding extra tile for contact journal and adding missing identifiers 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14281

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icons - ContactJournal.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "filename" : "cross_dark.pdf",
+      "idiom" : "universal"
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icons - ContactJournal.imageset/Contents.json]: Can we keep filename matching asset name?

**Review by flxschmidt — COMMENTED** [review]
LGTM! one minor thing

**Review by puneetmahali1 — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icons - ContactJournal.imageset/Contents.json (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Common/Icons - ContactJournal.imageset/Contents.json]: Done

**Review by flxschmidt — APPROVED** [review]

**Pull request merged**